src/style.css

```diff
@@ -2,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.rating-star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
Consider using a consistent unit for spacing throughout your CSS. If you're using pixels for widths and heights, it's a good practice to use pixels for margins as well for consistency.